pmm/pmm2-server-release.groovy

```diff
@@ -35,6 +35,9 @@ pipeline {
             defaultValue: '2.0.0',
             description: 'PMM2 Server version',
             name: 'VERSION')
+        string(
+            description: 'Password for PGP key',
+            name: 'PGP_KEY_PASS')
```

Comment:
Do you want to enter password via option? but it will be available for all users and can be  available in the log. Please use jenkins credentials for it.

Reply:
+1. @hors was faster :) it should be done via secrets